PowerPointLabs/PowerPointLabs/Converters/ColorPane/SelectedColorShiftLuminosityByFactor.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Drawing;
+using System.Windows.Data;
+
+using PowerPointLabs.ColorsLab;
+
+namespace PowerPointLabs.Converters.ColorPane
+{
+    [ValueConversion(typeof(HSLColor), typeof(string))]
+    class SelectedColorShiftLuminosityByFactor : IValueConverter
+    {
+        public static SelectedColorShiftLuminosityByFactor Instance = new SelectedColorShiftLuminosityByFactor();
+
+        public object Convert(object value, Type targetType, object parameter, System.Globalization.CultureInfo culture)
+        {
+            if (value == null)
+            {
+                Color color = (HSLColor)value;
```

Comment:
Was wondering if `value == null` then converting it to `HSLColor` wouldn't it be still null? Or is there a default value to the `HSLColor`?

Reply:
That's right, `HSLColor` when implicitly converted to `System.Drawing.Color` will handle `null` by setting the `Color` to `rgb(0, 0, 0)`. 

`Color` is a struct in C#, so not allowed to set as `null`. If I'm not wrong, there will be a `NullReferenceException` thrown if we attempt to set `Color` to `null`. But since we have our own `HSLColor`, this is handled here.